### PR TITLE
add script helpers

### PR DESCRIPTION
**Pull request by Bharath92**
closes https://github.com/Shippable/reqProc/issues/136

### CODE REVIEW

File: job/scriptsGen/generateScript.js [hunk 1]
```diff
@@ -96,6 +98,27 @@ function _getScriptHeader(bag, next) {
   );
 }
 
+function _getScriptHelpers(bag, next) {
+  var who = bag.who + '|' + _getScriptHelpers.name;
+  logger.verbose(who, 'Inside');
+
+  var headerFile = path.join(global.config.execTemplatesPath, 'job',
```
Comment (ric03uec) [inline job/scriptsGen/generateScript.js]: shouldn't this be helperFile

Comment (shrivara) [inline job/scriptsGen/generateScript.js]: `helperFile`?

File: job/scriptsGen/generateScript.js [hunk 2]
```diff
@@ -96,6 +98,27 @@ function _getScriptHeader(bag, next) {
...
+          'with err: %s', who, headerFile, err));
+        return next(err);
+      }
+      bag.taskScript = bag.taskScript.concat(header);
+      return next();
+    }
+  );
+}
+
+
```
Comment (shrivara) [inline job/scriptsGen/generateScript.js]: Extra new line.

File: job/scriptsGen/generateScript.js [hunk 3]
```diff
@@ -96,6 +98,26 @@ function _getScriptHeader(bag, next) {
...
 
+function _getScriptHelpers(bag, next) {
+  var who = bag.who + '|' + _getScriptHelpers.name;
+  logger.verbose(who, 'Inside');
+
+  var helperFile = path.join(global.config.execTemplatesPath, 'job',
+    bag.scriptHelpersFileName);
+
+  fs.readFile(helperFile, 'utf8',
+    function (err, header) {
```
Comment (shrivara) [inline job/scriptsGen/generateScript.js]: helper

**Pull request merged**